tests/Tunnelite.Tests/TunnelTests.cs

```diff
@@ -0,0 +1,31 @@
+using Xunit;
+
+namespace Tunnelite.Tests
+{
+    public class TunnelTests
```

Comment:
Consider implementing IDisposable pattern for the test class to ensure proper cleanup of tunnel resources after each test. You can use the xUnit IClassFixture interface for this purpose.